packages/account/src/wallet/wallet-unlocked.test.ts

```diff
@@ -49,18 +49,18 @@ describe('WalletUnlocked', () => {
   it('Sign a transaction using wallet instance', async () => {
     // #region wallet-transaction-signing
     // #import { WalletUnlocked, Signer };
+
     const provider = await Provider.create(FUEL_NETWORK_URL);
     const wallet = new WalletUnlocked(PRIVATE_KEY, provider);
```

Comment:
```suggestion
    const wallet = Wallet.fromPrivateKey(PRIVATE_KEY, provider);
```

Should we favour this instantiation method for users?

Reply:
Hmm as the constructor does not need to be private, I do not mind, however I do prefer the method so will update.